src/Mod/Start/StartPage/TranslationTexts.py

```diff
@@ -55,6 +55,10 @@ def get(handle):
     T_TEMPLATE_PARAMETRICPART_DESC = translate(
         "StartPage", "Create a part with the Part Design workbench"
     )
+    T_TEMPLATE_ASSEMBLY_NAME = translate("StartPage", "Assembly")
+    T_TEMPLATE_ASSEMBLY_DESC = translate(
+        "StartPage", "Create an Assembly with the Assembly workbench"
```

Comment:
s/Create an Assembly/Create an assembly/

Reply:
good point, at the time I just followed the format of the PartDesign template, but I guess that's only there due to the Part/PartDesign confusion, but that's not the case with assembly